keyboards/keycapsss/plaid_pad/keymaps/oled/keymap.c

```diff
@@ -0,0 +1,212 @@
+#include QMK_KEYBOARD_H
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
Added it. Thank you for the hint.

Reply:
Looks like it didn't get added?

Reply:
You are right. I forgot it and added it only to the glcdfont.c file.
Fixed it.